Documents/Letter_to_FDA.Rmd

```diff
@@ -14,9 +14,9 @@ Division of Data Management Services and Solutions
+I am writing to you at the recommendation of Paul Schuette on behalf of the [R Submissions Working Group](https://github.com/RConsortium/submissions-wg) which is organized under the R Consortium$^{1}$. Our working group, primarily composed of representatives from several different pharmaceutical companies, is engaged in an effort to explore and address challenges in adapting our regulatory submission process to accommodate submissions using the R programming language. We are part of a larger effort to facilitate the use of R in the regulatory process that includes the R Consortium working groups: the [R Validation Hub](https://www.pharmar.org/) and [R Tables for Regulatory Submissions](https://github.com/RConsortium/rtrs-wg), and the PhUSE working group [Clinical Statistical Reporting in a Multilingual World](https://phuseaccount.sharepoint.com/sites/ClinicalStatisticalReportinginaMultilingualWorld/_layouts/15/Doc.aspx?sourcedoc={3334b95b-6ac7-4feb-bfe4-cb154dac4575}&action=edit&wd=target%282021%20Meeting%20Minutes.one%7Cbf68bb6a-f925-4411-b285-f79a4705c140%2FJanuary%2011th%7C3fe896ca-003e-44ea-b31e-d8d3877844bb%2F%29). As well, this work is in alignment with the Transcelerate [MSA framework](https://www.transceleratebiopharmainc.com/wp-content/uploads/2021/04/MoA-Initiative_MSAFramework_April-2021.pdf) by pioneering the regulatory delivery of analysis using more rapidly developed, open source tools.
 
-We are presently working on a pilot submission to demonstrate that it is possible to make all R submissions in a manner that is consistent with the FDA's infrastructure and processes. Because this project is a multi-company, collaborative project being undertaken on behalf of the Pharmaceutical industry, we would like the R Consortium to register as the sponsor for the pilot submission.
+We are presently working on a pilot submission to demonstrate that it is possible to make all R submissions in a manner that is consistent with the FDA's infrastructure and processes. Because this project is a multi-company, collaborative project being undertaken on behalf of the Pharmaceutical industry, we would like the explore the possibility of registering the R Consortium as a sponsor for the pilot submission. By registering a public and cross-pharma group as a sponsor, we hope to develop pilots in the open, without requiring an individual industry sponsor to manage a pilot submission.
```

Comment:
```suggestion
We are presently working on a pilot submission to demonstrate that it is possible to make all R submissions in a manner that is consistent with the FDA's infrastructure and processes. Because this project is a multi-company, collaborative project being undertaken on behalf of the Pharmaceutical industry, we would like to explore the possibility of registering the R Consortium as a sponsor for the pilot submission. By registering a public and cross-pharma group as a sponsor, we hope to develop pilots in the open, without requiring an individual industry sponsor to manage a pilot submission.
```